bpf/bpf_host.c

```diff
@@ -25,6 +25,17 @@
 # undef ENABLE_HOST_FIREWALL
 #endif
 
+/* Controls the inclusion of the CILIUM_CALL_HANDLE_ICMP6_NS section in the
+ * bpf_lxc object file.
+ */
+#define COMPILE_ICMPV6_NS_HANDLING
+
+/* Controls the inclusion of the CILIUM_CALL_SEND_ICMP6_TIME_EXCEEDED section
+ * in the bpf_lxc object file. This is needed for all callers of
+ * ipv6_local_delivery, which calls into the IPv6 L3 handling.
+ */
+#define COMPILE_ICMPV6_HOPLIMIT_HANDLING
```

Comment:
Too bad loader cannot detect this right now :/ In XDP & sockmap we've been using `SKIP_*` defines so far. Maybe makes sense to be consistent and also use `SKIP_` prefix for the ICMPv6 code here?

Reply:
> Too bad loader cannot detect this right now :/

I considered extending our iproute2 fork to check if those sections are used before importing them but it's quite a change given it doesn't currently walk the bytecode.

> Maybe makes sense to be consistent and also use SKIP_ prefix for the ICMPv6 code here?

Ah, I missed that. Makes sense, I fixed it.